backstory/src/demo/webapp/gpt2.js

```diff
@@ -11,9 +10,78 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+/* eslint-disable no-unused-vars */
+
+/**
+ * Sends request to text generation servlet and displays
+ * result to front end.
+ */
+async function displayStory() {
+  const input = document.getElementById('prompt').value;
+
+  if (input === '' || input === null) {
+    alert('You need to enter a value');
+    return;
+  }
+
+  console.log(input);
+
+  // get display
+  const display = document.getElementById('story-display');
+  display.appendChild(formatResponse('Generating... Please be patient.'));
+  // grab data and get its text version (it is sent as JSON)
+  const response = await fetch('/gpt2', {
+    method: 'post',
+    headers: {'Content-Type': 'application/json'},
+    body: `{text: ${input}}`,
+  });
+
+  
+  const data = await response.text();
+
+  display.innerHTML = data;
+
+  // parse the JSON into an object
+  const jsonObject = JSON.parse(data);
+
+  console.log(jsonObject);
```

Comment:
Did you mean to leave this log statement in? It's ok if you did, but if it was just there for debugging, maybe take it out.

Reply:
Done